### DISCUSSION TITLE
Can I use the GraphQL API with a GitHub App?

### DISCUSSION THREAD

**Discussion by atarithelarge** [community/community#24909] [Apps, API and Webhooks]
<p>It’s all in the subject line: Can I use the GraphQL API with a GitHub App?</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Yes! GitHub Apps now fully support the GraphQL API v4. <a href="https://developer.github.com/changes/2018-04-30-graphql-supports-github-apps/" rel="nofollow noopener">Check out the</a><a href="https://developer.github.com/changes/2018-04-30-graphql-supports-github-apps/" rel="nofollow noopener">announcement</a><a href="https://developer.github.com/changes/2018-04-30-graphql-supports-github-apps/" rel="nofollow noopener">from April</a> for more details.</p>
